org.sugarlabs.Pointillism.json

```diff
@@ -0,0 +1,254 @@
+                    "sha256": "d16a0141ec1a18405cd4ce8b4613101da75da0e9a7aec5bdd4fa804d0e0eba73"
+                }
+            ]
+        },
+        {
+            "name": "telepathy-glib",
+            "config-opts": ["--disable-static", "--disable-gtk-doc"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://www.github.com/TelepathyIM/telepathy-glib.git",
+                    "tag": "telepathy-glib-0.24.1",
+                    "commit": "c834ef1a50a97cbfae7a0d560ff806fd5aa7ffe8"
+                },
+                {
+                    "type": "patch",
+                    "path": "telepathy-glib-uniquify.patch"
+                }
+            ]
+        },
+        {
+            "name": "sugar-toolkit-gtk3",
+            "no-parallel-make": true,
+            "config-opts": [
+                "--with-python3"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.sugarlabs.org/sources/sucrose/glucose/sugar-toolkit-gtk3/sugar-toolkit-gtk3-0.116.tar.xz",
+                    "sha256": "745a360577efceb10f57ca4cbad5f9ec61f7ed6cc381eeb72422bbd7455b770b"
+                }
+            ]
+        },
+        {
+            "name": "sugarapp",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links ${PWD} --install-option=\"--prefix=${FLATPAK_DEST}\" $PWD"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/tchx84/sugarapp.git",
+                    "tag": "v1.10",
+                    "commit": "54b0efdc0bbef60d48685aa2a1559a3fb8f2ddf5"
+                }
+            ]
+        },
+        
+        
+        {
+            "name": "pointillism",
+            "buildsystem" : "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST} --skip-install-desktop-file --skip-install-mime"
+            ],
+            "sources" : [
+                {
+                    "type": "git",
+                    "url": "https://github.com/sugarlabs/pointillism-activity",
+                    "commit": "ca8068cb39155a50f88c6458dbd42242bc150fd3"
+                },
+                {
+                    "type": "patch",
+                    "path": "sugargame.canvas.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pointillism-port.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "pointillism-info.patch"
+                }
+            ],
+            "post-install": [
+                "sugarapp-gen-mimetypes activity/activity.info mimetypes",
```

Comment:
Since this app does not save/restore states, no need for generating mimetypes file. Also remove other references to this file below.